### ISSUE TITLE
bug report: teamsAppExtendToM365.UnhandledError

### ISSUE THREAD

**Issue by zmnxcb** [OfficeDev/TeamsFx#8851]
An unexpected error has occurred while performing the <REDACTED: user-file-path> task. The reason for this error is: Unexpected token 'j', "j�
�П��\�

**Comment by kuojianlu** [comment]
@zmnxcb This is a known issue related to login. Followings are the workaround that may unblock you.

1. Sign out the M365 account, and then re-sign in. After that, retry F5.
2. If the above method cannot solve this error, comment out `teamsApp/extendToM365` action in `teamsapp.local.yml`. After that, retry F5. (Please note that this workaround is not applicable to the scenario that you debug in Outlook or the Microsoft 365 app) ![image](https://user-images.githubusercontent.com/37978464/240117238-e56087fe-a5c1-4351-86cf-135d5c1a8c9a.png)

**Comment by kuojianlu** [comment]
Duplicate of https://github.com/OfficeDev/TeamsFx/issues/8741.
